pkg/testing/integration/util.go

```diff
@@ -30,6 +30,26 @@ import (
 	"github.com/pulumi/pulumi/pkg/util/contract"
 )
 
+// DecodeMapString takes a string of the form key1=value1:key2=value2 and returns a go map.
+func DecodeMapString(val string) (map[string]string, error) {
```

Comment:
This doesn't appear to be used?

Reply:
It's a utility function I'll want to use in some repositories to compute the set of overrides to add based on information from NPM.  I wanted to centralize that logic, so I put it here.